### ISSUE TITLE
Allow using iPhone to gather data (Airport Utility + Shortcut to send data to website)

### ISSUE THREAD

**Issue by ireuben** [hnykda/wifi-heatmapper#1]
Would be sweet to be able to use iPhone to walk around and make the measurements. But the only app that Apple allows to have access to wifi info is Airport Utility.

This project https://github.com/kavinaidoo/ioswifisurvey is a pretty cool workaround for that, which uses a Shortcut so that in the Airport Utility you can do a wifi survey, then click Share and pick the shortcut, and it sends the wifi data to a website. Perhaps wifi-heatmapper could use a similar shortcut, and when it launches the url (with the wifi data included) you could pick the spot on the floorpan where you are, and it could add all the wifi signal strength data to that location.

**Comment by hnykda** [comment]
Yeah, thanks for the suggestion. Would be awesome to have something like that, I was thinking about the same with my android phone. I don't think the current architecture allows for that though, unfortunately. It's trivial to host this app on some address, and connect to it, but the `iperf` measurements would happen to be on the machine this app is running.

However, this app is relatively agnostic to where the data come from - as long as it follows fairly simple JSON structure, you can visualize it. I think it would be easier to rather produce some output like that e.g. using the similar approach as above and use that as a database file then somehow figuring out a way how to get around this. 

I am quite annoyed how bad Apple is with allowing things like these to happen... For example, to the best of my knowledge, there is no way how to force 5ghz over 2ghz on Mac (which would be useful for the measurements here, when you wanted to really see how far you are). Unless you have two different SSIDs, you are out of luck 😡 .
